scipy/spatial/_spherical_voronoi.py

```diff
@@ -261,7 +261,12 @@ def sort_vertices_of_regions(self):
         """
         if self._dim != 3:
             raise TypeError("Only supported for three-dimensional point sets")
-        self.regions = _voronoi.sort_vertices_of_regions(self._simplices, self.regions)
+        tmp = _voronoi.sort_vertices_of_regions(self._simplices, self.regions)
+        # the Cython version modifies self.regions in-place, 
+        # while the Pythran version works on a deep copy of input argument
+        # so if return is None, it means you are calling the Cython one
+        if tmp != None:
```

Comment:
It is a small point compared to the test failures that remain, but I believe `if tmp is not None` is perhaps slightly preferred, and a nudge faster sometimes i.e., https://stackoverflow.com/a/3257957/2942522

Reply:
And it's more *idiomatic*